pkg/edgexsdk/edgexsdk.go

```diff
@@ -301,6 +317,7 @@ func (sdk *AppFunctionsSDK) listenForInterrupts() {
 
 		signalReceived := <-signalChan
```

Comment:
For consistency, change "signalChan" to "signals". Yea, I know I created that one... ;-)